File: setled.py
#!/usr/bin/python
import json
from sense_emu import SenseHat
print('IN')

sense = SenseHat()

filename = "leddata.json";

if filename:
    with open(filename, 'r') as f:
        ledDisplayArray=json.load(f);
        
for led in ledDisplayArray:
    # schemat led: y x R G B
    sense.set_pixel(led[1], led[0], led[2], led[3], led[4]);

File: setled.php
#!/usr/bin/php
<?php
	ini_set('display_errors',1);
	error_reporting(E_ALL);
	
	
	function ledIndexToTagConverter($x, $y){
		return "LED" .$x .$y;
	}
	
	$ledDisplay = array();
	$ledDisplayDataFile = 'leddata.json';
	
	$n=0;
	
	for ($i=0; $i<8; $i++){
		for ($j=0; $j<8; $j++){
			$ledTag=ledIndexToTagConverter($i, $j);
			if(isset($_POST[$ledTag])){
				$ledDisplay[$n] = json_decode($_POST[$ledTag]);
				$n=$n+1;
			}
		}
	}
	
	$ledDisplayJson=json_encode($ledDisplay);
	$dataFile = fopen($ledDisplayDataFile, 'w+') or die("ERR1");
	fwrite($dataFile, $ledDisplayJson);
	fclose($dataFile);
	
	echo "ACK1 ";
	$command = escapeshellcmd('sudo python setled.py');
	$output = shell_exec($command);
	echo $output;
	echo "ACK2 ";
?>